content/posts/2018-10-25--.net-in-my-browser.md

```diff
@@ -0,0 +1,345 @@
+---
+title: .NET in MY browser?
+subTitle: Blazor - a WASM powered front-end framework
+tags: ["c#", "razor", "blazor", "wasm", "web", "front-end", "asp.net core", "mono"]
+cover: /img/dotnet.png
+postAuthor: Krzysztof Miczkowski
+---
+
+# Introduction
+
+.NET ecosystem has seen some rapid changes in the recent times. With advent of [.NET Core](https://dot.net/core), the official cross-platform implenetation of the runtime (and SDK) Linux and Mac users have gained a way to write and run .NET code in the comfort of their favourite OSes, all supported and ratified by Microsoft. The new and shiny tooling as well as ASP.NET Core's impressive performance (in top 10 of TechEmpower's [Fortunes benchmark](https://www.techempower.com/benchmarks/#section=data-r16&hw=ph&test=fortune)) have overshadowed .NET Core's older x-plat sibling, the Mono runtime. But it is one of Mono's targets that interests us today - namely WebAssembly. Ladies and gentlemen, allow me to introduce...
```

Comment:
implenetation -> implementation

Reply:
Gaah! Typo in the first sentence >.< Thanks, good catch.